### PR TITLE
feat: icky 1150 icky-1151 b2b sbma implementation

### PR DESCRIPTION
**Pull request by Kibo-TriptiV**

### CODE REVIEW

**Review by kibo-chandradeeptalaha — APPROVED** [review]

File: public/locales/en/common.json [hunk 1]
```diff
@@ -531,5 +531,6 @@
   "menu": "Menu",
   "remove-payPal": "Remove PayPal",
   "loading-paypal": "Loading PayPal...",
-  "error-cartTakeover": "Error while fetching the cart. Please close the window and try again."
+  "error-cartTakeover": "Error while fetching the cart. Please close the window and try again.",
+  "accounts": "Accounts"
```
Comment (kibo-sushant) [inline public/locales/en/common.json]: Add corresponding keys in other local file as well

File: src/pages/api/graphqlWUC.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { graphQLWithoutUserClaimsHandler } from '@/lib/api/handlers'
+import withLogger from '@/lib/api/util/with-logger'
+
+export default withLogger(graphQLWithoutUserClaimsHandler as any)
```
Comment (kibo-sushant) [inline src/pages/api/graphqlWUC.ts:4]: Do we need this? Kevin had suggested to check with Commerce team

File: lib/api/handlers/switch-user.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import { NextApiRequestWithLogger } from '@/lib/types'
+
+async function switchUserHandler(req: NextApiRequestWithLogger, res: NextApiResponse) {
+  try {
+    const { id } = req.query
+
+    const refreshAuthTicketsResponse = await refreshCustomerAuthToken(
+      id as string,
+      req as NextApiRequestWithLogger
+    )
```
Comment (kibo-sushant) [inline lib/api/handlers/switch-user.ts:22]: Ideally, we should not call operations from handler

**Review by kibo-sushant — APPROVED** [review]

File: public/locales/en/common.json (hunk 1 above)
Comment (Kibo-TriptiV) [reply public/locales/en/common.json]: we are maintaining en file only.

File: src/pages/api/graphqlWUC.ts (hunk 2 above)
Comment (Kibo-TriptiV) [reply src/pages/api/graphqlWUC.ts:4]: yes right now we need this to make it run. parallelly we are connected with COM team.

File: lib/api/handlers/switch-user.ts (hunk 3 above)
Comment (Kibo-TriptiV) [reply lib/api/handlers/switch-user.ts:22]: done

**Pull request merged**